tests/dummy/app/foo/bar/baz/show-with-params/route.js

```diff
@@ -0,0 +1,32 @@
+import Ember from 'ember';
+
+const get = Ember.get;
+const set = Ember.set;
```

Comment:
After merging in #37, you can destructure `get` and `set` inside the one on L5-7